packages/plugin-serverless/src/commands/serverless/env/get.js

```diff
@@ -0,0 +1,41 @@
+const { TwilioClientCommand } = require('@twilio/cli-core').baseCommands;
+
+const {
+  handler,
+  cliInfo,
+  describe,
+} = require('twilio-run/dist/commands/env/env-get');
+const {
+  convertYargsOptionsToOclifFlags,
+  normalizeFlags,
+  createExternalCliOptions,
+  getRegionAndEdge,
+} = require('../../../utils');
+
+const { flags, aliasMap } = convertYargsOptionsToOclifFlags(cliInfo.options);
+
+class EnvGet extends TwilioClientCommand {
```

Comment:
There is a lot of copy and paste! Any merit in creating a `TwilioRunCommand` that normalizes the flags, gets the region and edge, turns them all into `opts` and `externalOpts` and passes off to the `handler`? Or even just a factory function for this?

Reply:
Yeah it's a good point! I'll do that